plugins/embark-slither/src/index.ts

```diff
@@ -0,0 +1,40 @@
+import fs from "fs-extra";
+import shelljs from "shelljs";
+
+function isSlitherInstalled() {
+  return shelljs.which("slither");
+}
+
+function executeSlither(path: string) {
+  shelljs.exec(`slither ${path}`);
+}
+
+function getHeader(path: string) {
+  return `
+JSON AST (compact format):
+
+
+======= ${path} =======
+
+`;
+}
+
+function run(compilationResult: { sources: { [path: string]: { ast: object }}}) {
```

Comment:
Personally I feel like putting it all in one line instead of extracting a `type` is harder to read.